dashboard/lib/policies/active_record_roles.rb

```diff
@@ -1,27 +1,29 @@
 # This module contains policies relating to the roles used by ActiveRecord to
 # handle multiple database connections
-module Policies::ActiveRecordRoles
-  # Get the name of ActiveRecord "role" used for connecting to the writer
-  # database, as configured in database.yml.
-  def self.get_writing_role_name
-    :primary
-  end
+module Policies
+  module ActiveRecordRoles
```

Comment:
This is necessary to avoid `uninitialized constant Policies::ActiveRecordRoles (NameError)` we otherwise get when trying to directly require this code path without the aid of the autoloder